test/Core.Test/AdminConsole/OrganizationFeatures/Policies/PolicyRequirements/PersonalOwnershipPolicyRequirementTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿using AutoFixture.Xunit2;
+using Bit.Core.AdminConsole.Enums;
+using Bit.Core.AdminConsole.Models.Data.Organizations.Policies;
+using Bit.Core.AdminConsole.OrganizationFeatures.Policies.PolicyRequirements;
+using Bit.Core.Enums;
+using Bit.Core.Test.AdminConsole.AutoFixture;
+using Xunit;
+
+namespace Bit.Core.Test.AdminConsole.OrganizationFeatures.Policies.PolicyRequirements;
+
+public class PersonalOwnershipPolicyRequirementTests
+{
+    [Theory, AutoData]
+    public void DisablePersonalOwnership_IsFalse_IfNoPersonalOwnershipPolicies(
```

Comment:
Maybe we should follow the `should-when` naming pattern for consistency?

Consider: `DisablePersonalOwnership_ShouldBeFalse_WhenNoPersonalOwnershipPoliciesExist`